docs/README.md

```diff
@@ -0,0 +1,487 @@
+---
```

Comment:
We had that mess also in the current setup, so I manually copy / pasted them. Having an automated sync would be great, but we could do that as follow up

Reply:
Added a todo for this. It would be straight forward to add a copy step as part of the build, so I can do that. I'd like to see the landing page(s) / links (etc) in their final state (if we change them substantially, nbd if not) before doing it, just in case it is decided to do something different with the content.